docs/config/services/deployments/production.js

```diff
@@ -39,7 +39,8 @@ module.exports = function productionDeployment(getVersion) {
       'components/lunr.js-' + getVersion('lunr.js') + '/lunr.min.js',
       'components/google-code-prettify-' + getVersion('google-code-prettify') + '/src/prettify.js',
       'components/google-code-prettify-' + getVersion('google-code-prettify') + '/src/lang-css.js',
-      'js/versions-data.js',
+      'js/version-data.js',
+      'https://code.angularjs.org/snapshot/docs/js/versions-data.js',
```

Comment:
note that in production we read the list from snapshot
otherwise we just use the local